### PR TITLE
Update to jenkins

### PR DESCRIPTION
**Pull request by batmat**
We need a more recent release, also because the latest release still depends on CVS...

### CODE REVIEW

**Comment by batmat** [comment]
Hi @dty, do you still plan/want to maintain that plugin, or do you want me to take over maintenance?

Thanks!

**Comment by batmat** [comment]
I thought about that, and then as I wanted to use a 2.x parent pom I thought it would be simpler to just bump it to some place I'm sure it'd work easily. Also because I'm not changing anything to the code anyway, existing user could:
* either stay where they are if their Jenkins version is too old
** because since now some time anyway, Jenkins will filter out the Update Center updates with the Jenkins version you're running. IOW, anyway users of old Jenkins won't even be bothered by a new incompatible release because they shouldn't see it.
* or update, simply

**Comment by batmat** [comment]
@dty Thanks! So as discussed, I'm going to merge this soon then:

If you can re-confirm here or somewhere else publicly that you're OK for me to take over maintenance, I'll then file the necessary PR against github.com/jenkins-infra/repository-permissions-updater/ to be able to release.

File: src/main/java/hudson/plugins/extendedread/PluginImpl.java [hunk 1]
```diff
@@ -28,7 +27,7 @@ private void enablePermission() throws Exception {
         try {
             Class clazz = Item.class;
             if (clazz!=null) {
-                Field f = clazz.getDeclaredField("EXTENDED_READ");
+                clazz.getDeclaredField("EXTENDED_READ");
```
Comment (jtnord) [inline src/main/java/hudson/plugins/extendedread/PluginImpl.java]: This is available in 1.609 - should this not be yanked?

**Comment by batmat** [comment]
Will release a new version tomorrow or so. Will need to file a PR on the permission repo for that purpose BTW.

Thanks @dty and @jtnord !

**Pull request merged**